class.jetpack-autoupdate.php

```diff
@@ -15,19 +20,25 @@ static function init() {
 	}
 
 	private function __construct() {
-		$this->updates_allowed = Jetpack_Options::get_option( 'json_api_full_management', false );
+		$this->updates_allowed = Jetpack::is_module_active( 'manage' );
 
```

Comment:
A comment here wouldn't go amiss -- explaining that the code will only auto update plugins that the user has explicitly opted in to auto updating through the wpcom ui.